log-insights/setup/syslog-server.mdx

```diff
@@ -5,9 +5,40 @@ backlink_title: 'Log Insights: Setup'
 ---
 
 If your database logs are available via syslog, you can use the [`db_log_syslog_server`](/docs/collector/settings#self-managed-servers)
-setting to have the collector listen for syslog messages from a remote server. This should be a local address like
-"0.0.0.0:32514" and it's recommended to use an unprivileged port number (1024 and up) to avoid running
-the collector as root.
+setting to have the collector listen for syslog messages from a remote server.
+When this is specified, the collector will run the syslog server behind the
+scene on the specified port to receive syslog messages.
+This should be a local address like "0.0.0.0:32514" and it's recommended to use
+an unprivileged port number (1024 and up) to avoid running the syslog server as
+root.
 
-Note that you need to use protocol RFC5424, with an unencrypted TCP connection. Due to syslog not being
-an authenticated protocol it is recommended to only use this integration over private networks.
+Note that you need to use protocol RFC5424, with a TCP connection. When not
+using TLS, it is recommended to only use this integration over private networks.
```

Comment:
I feel like this waters down this warning a bit. If you're using TLS but _not_ using client certs, someone could still be sending bogus logs to your collector and you won't be able to tell. Now, the likelihood of this happening is pretty small, and the damage they can do is pretty limited (DoS your collector? cause bogus information to show up in your Log Insights page?), but I think I'd still rather make it clear what the situation is and allow the user to decide whether they want to take that chance.

Reply:
Yeah fair point. I'll tweak to cover that

Reply:
Uhmm while trying to tweak this, I started wondering about our choice `VerifyClientCertIfGiven` https://pkg.go.dev/crypto/tls#ClientAuthType

You're right that the using TLS won't be totally secure, in terms of anyone can send out logs to the collector (still secure in terms of eavesdropping and tampering). When someone actually wants to ensure the mTLS (aka prevent from any random folks sending out logs), they can't really do this with `VerifyClientCertIfGiven`, because we only verify when the client cert is given, so bad person can still send out logs without the client cert and the collector will just receive it.
I think `VerifyClientCertIfGiven` is still valid as a default behavior, though we might consider supporting additional options if there's a request for such a feature.

Reply:
Update, maybe I'd love to have one more look this part (no rush).

Reply:
Gonna merge as is but feel free to leave any comments if you have, happy to create a follow-up PR

Reply:
The revised text looks good. Good point on `VerifyClientCertIfGiven`. I think you're right; we can address that if it comes up.